### DISCUSSION TITLE
Windows Vista graphics acceleration

### DISCUSSION THREAD

**Discussion by kolo2019** [utmapp/UTM#6143] [Windows]
Is there any way to get accelerated graphics in Windows Vista? I'm not looking to play games, but I want to use programs such as Windows Media Center

UTM Version: 4.4.4
macOS Version: Ventura 13.2.1
Mac Chip: M1

Virtual machines OS is Windows Vista Ultimate

The settings are:
architecture x86_64 
Standard PC (i440FX + PIIX, 1996) (alias of pc-i1440fx-7.2) (pc)
RAM 2GB

**Comment by lgrachov** [comment]
Unfortunately, UTM only supports graphic acceleration for Linux and macOS virtual machines. Try using another virtualisation app.
